composer.json

```diff
@@ -13,6 +13,7 @@
     }
   ],
   "require": {
+    "php": "^7.1",
     "sylius/sylius": "^1.0@beta",
```

Comment:
since interfaces changed on sylius 1.0.0, we can not be compatible with both beta and 1.0.0
so, please update sylius minimum version, and move it to ^1.0.0  

Then I will create a new version. old version will keep serving beta. new one is going to be for ^1.0.0

Reply:
@Snake-Tn Makes sense. Commit updating sylius/sylius to 1.0.0 is on the way.